app/src/main/java/com/example/una/BugApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.example.una;
+
+import android.app.Application;
+
+import com.buglife.sdk.Buglife;
+import com.buglife.sdk.InvocationMethod;
+
+public class BugApplication extends Application {
```

Comment:
Actually,  this is your application as you specified it in the manifest. Call it `UnaApplication`